slack_gpt_bot.py

```diff
@@ -82,14 +117,15 @@ def command_handler(body, context):
         beta-slack-chatgpt-bot (channel_id: C057NBLL2G4). If the message
         didn't originate from this channel, you got a polite message and 
         were denied access.
+        
+        if channel_id != 'C057NBLL2G4': #lock to test channel for beta
+            slack_resp = app.client.chat_postMessage( 
+                channel=channel_id,
+                thread_ts=thread_ts,
+                text="Our apologies, however the Beta ChatGPT bot is not allowed outside of the beta-slack-chatgpt-bot channel"
+            )
+            return
```

Comment:
Do we want to add the channel lock back again?

Reply:
No, that was for our beta release back in May. I should remove the comment as it is preserved by Github if we ever needed to know what we did 😄